executor/Dockerfile

```diff
@@ -25,5 +25,8 @@ RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 GO111MODULE=on go build -a -o executor
 FROM gcr.io/distroless/static:latest
 WORKDIR /
 COPY --from=builder /workspace/executor .
+# Copy openapi spec and swagger UI files
+COPY ../openapi/swagger-ui/ /workspace/api/rest/openapi/
+COPY ../openapi/seldon.json /workspace/api/rest/openapi/seldon.json
```

Comment:
Does the `seldon.json` file exist in the `openapi` root folder? I can only see it under `python/seldon_core`.